server/integration_tests/test_data/demo_feb2023/query_4/chart_config.json

```diff
@@ -526,7 +526,7 @@
             "statVar": "Count_Worker_NAICSRealEstateRentalLeasing"
           },
           "Count_Worker_NAICSServiceProviding": {
-            "name": "Population of Person: Service-providing (NAICS/102)",
+            "name": "Population: Service-producing",
```

Comment:
Probably too late to change now, but we this should be `Workers: Service-producing`.

Reply:
yes, these were added by Prem: https://screenshot.googleplex.com/8hToSxcawnY8hsH